Tests/DependencyInjection/Compiler/TwigFormPassTest.php

```diff
@@ -5,6 +5,9 @@
 use Norzechowicz\AceEditorBundle\DependencyInjection\Compiler\TwigFormPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
+/**
+ * @author azzra <azzra@users.noreply.github.com>
```

Comment:
Yes but I did created it, I forgot to put it the first time :)
So what about  a new PR for fixing author annotation?
Even if Github show authors, I -personnaly- think it's a good idea to write down authors in header files. If it goes againts contributing in this project, no problem but let's sort out this in one dedicated PR, no ?